web-client/src/presenter/computeds/caseDetailHeaderHelper.js

```diff
@@ -48,11 +48,17 @@ export const caseDetailHeaderHelper = (get, applicationContext) => {
     }
   }
 
-  const showConsolidatedCaseIcon = !!caseDetail.leadCaseId;
+  const showConsolidatedCaseIcon = !!caseDetail.leadDocketNumber;
 
   const showCreateOrderButton = permissions.COURT_ISSUED_DOCUMENT;
 
-  const showAddDocketEntryButton = permissions.DOCKET_ENTRY;
+  const hasDocketEntryPermission = permissions.DOCKET_ENTRY;
```

Comment:
perhaps added for debugging, but it doesn't live very long

Reply:
Good catch!

Reply:
Updated!